composer/diagram/package.json

```diff
@@ -2,7 +2,11 @@
   "name": "@ballerina/diagram",
   "version": "0.982.1-SNAPSHOT",
   "description": "Provides a set of react components to draw Ballerina Diagrams.",
-  "keywords": ["ballerina", "ballerina-lang", "composer"],
+  "keywords": [
```

Comment:
Seems like these changes are not related to this PR.

Reply:
Reverted